src/components/button/template.njk

```diff
@@ -1,18 +1,39 @@
-{# Determine type of element to use, if not explicitly set -#}
+{# Set classes for this component #}
+{%- set classNames = "govuk-button" -%}
 
-{% if params.element %}
+{%- if params.classes %}
+  {% set classNames = classNames + " " + params.classes %}
+{% endif %}
+{% if params.disabled %}
+  {% set classNames = classNames + " govuk-button--disabled" %}
+{% endif -%}
+
+{# Determine type of element to use, if not explicitly set #}
+{%- if params.element %}
   {% set element = params.element | lower %}
 {% else %}
   {% if params.href %}
     {% set element = 'a' %}
   {% else %}
     {% set element = 'button' %}
   {% endif %}
+{% endif -%}
+
+{% if params.isStartButton %}
+  {% set iconHtml %}
+{#- The SVG needs `focusable="false"` so that Internet Explorer does not
+treat it as an interactive element - without this it will be
+'focusable' when using the keyboard to navigate. #}
+<svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" focusable="false">
```

Comment:
Non-blocking: It would be nice to indent the svg similarly to https://github.com/alphagov/govuk-frontend/blob/840d019e0412309d6f8ab1032ec3b17b30fcd955/src/components/header/template.njk#L20 in the template for readability

Reply:
This was done intentional because nunjucks was not producing neat example markup in the review app.